blog/03-07-2-23-building-a-celo-remittance-system-a-technical-tutorial-for-developers/building-a-celo-remittance-system-a-technical-tutorial-for-developers.md

```diff
@@ -0,0 +1,339 @@
+          "https://forno.celo.org"
+        )
+      },
+      network_id: 42220,
+      gas: 10000000
+    }
+  },
+  compilers: {
+    solc: {
+      version: "0.8.7",
+      settings: {
+        optimizer: {
+          enabled: false,
+          runs: 200
+        },
+        evmVersion: "istanbul"
+      }
+    }
+  }
+};
+```
+
+This code configures Truffle to use the Celo network, specifically the ‘alfajores-forno’ testnet. Replace the URL with the URL of the Celo network you want to use If you are deploying to the Celo mainet.
+
+Now, let's deploy the smart contract to the Celo network. In your terminal, run the following command:
+
+```bash
+truffle deploy --network alfajores
+```
+
+This would deploy directly to the alfajores network.
+
+You might get an error like this:
+
+![image](./images/7.png)
+
+The error indicates there are not sufficient funds in the contract address, this address should be funded. You can either fund this address by sending funds to it from the testnet faucet or metamask.
+
+Running the "truffle deploy --network alfajores" command again, will deploy the smart contract to the Celo network, and you should see an output similar to this:
+
+![image](./images/8.png)
+
+Note the contract address for the deployed Remittance contract. We will need this address to interact with the contract on the Celo network.
+
+With the contract deployed, we can build a React application to interact with our remittance system.
+
+## Conclusion
+
+In this tutorial, we covered how to build and deploy a remittance system on the Celo network using smart contracts and Truffle. By finishing this tutorial, you have gained technical skills and knowledge in:
+
+Setting up a development environment for Celo blockchain development.
+Writing and deploying smart contracts on the Celo network using Solidity and Truffle.
+Implementing secure messaging protocols to enhance the security of the remittance system. Congratulations on successfully building a remittance system on the Celo network!
+
+## Next Steps
+
+Now that you have completed this tutorial, here are some ideas for continued learning and exploration. You can experiment with different user interfaces for your remittance system, such as building a mobile app or a web-based interface. You can also explore adding more features to the remittance system, such as transaction tracking, user authentication, or automatic conversion to local currencies.
+
+It is strongly recommended that as you build on Celo, you check out Celo's documentation, and join the Celo Developer Discord community to connect with other developers.
+
+## About the Author
+
+Oluwalana is a technical writer and a generalist. As an analog-by-birth and digital-by-nature individual, Oluwalana leverages his unique perspective to craft compelling stories and thought-provoking content.
+
+## References
+
+(https://www.investopedia.com/terms/r/remittance.asp)
```

Comment:
[investopedia](https://www.investopedia.com/terms/r/remittance.asp),like these we are supposed to add the links in the markdown file

Reply:
Done